Gradle.md

```diff
@@ -8,9 +8,47 @@ There are currently five experiments for Gradle. You could also perform these ex
+
+### User permissions
+
+Experiment builds running locally require different permissions than those that run in CI.
+Note that while not all permissions are required, they're still strongly recommended for the best possible experience.
+
+Your Develocity user should be granted the following permissions:
+
+| User permission                 | Reason                                                                                                                                      | Required              |
+|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|-----------------------|
+| Publish build scans             | Analyzing the experiment results relies on build scans published to Develocity.                                                             | Yes                   |
+| Read build cache data           | Reading from the remote build cache.                                                                                                        | Only for experiment 5 |
+| View build scans and build data | Build scans can be used to perform a deeper analysis of the experiment results.                                                             | No                    |
+| Access build data via the API   | Used to retrieve the summarized experiment results. See [Authenticating with Develocity](#authenticating-with-develocity) for more details. | No                    |
+
+> [!IMPORTANT]
+> Your Develocity user must be granted one of `View build scans and build data` or `Access build data via the API`.
+> If you have neither, you will have no way to view the experiment results.
+
+The Develocity user used for CI builds should be granted the following permissions:
+
+| User permission                 | Reason                                                                                                                                      | Required                                        |
+|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------|
+| Publish build scans             | Analyzing the experiment results relies on build scans published to Develocity.                                                             | Yes                                             |
+| Read and write build cache data | Writing to the remote build cache.                                                                                                          | Yes                                             |
+| Access build data via the API   | Used to retrieve the summarized experiment results. See [Authenticating with Develocity](#authenticating-with-develocity) for more details. | Only when using `--fail-if-not-fully-cacheable` |
```

Comment:
I find the last sentence a bit 'confusing'.

Reply:
Which sentence specifically do you find confusing?

Reply:
`Only when using '--fail-if-not-fully-cacheable'`. I would at least repeat that the permission is mandatory when running with the mentioned flag.